README.md

```diff
@@ -1,18 +1,25 @@
-# harmony-gdal
+# harmony-gdal-adapter
 
 ![](https://data-services-github-badges.s3.amazonaws.com/cov.svg?dummy=true)
 
-<img src="https://data-services-github-badges.s3.amazonaws.com/cov.svg?dummy=hello" />
+A demonstration of a subsetter capability to be used with Harmomy.
 
-A demonstration of a subsetter capability to be used with Harmomy. Deployed on [dockerhub](https://hub.docker.com/repository/docker/asfdataservices/gdal-subsetter/general)
+Deployed on [ghcr.io](https://github.com/nasa/harmony-gdal-adapter/pkgs/container/harmony-gdal-adapter) GitHub's Container registry.
 
-## Installing dependencies - make sure to clone harmony-service-lib-py locally
-pip3 install ../harmony-service-lib-py/ --target deps/harmony
 
 ## Build image
 bin/build-image
 
-## To run tests
+## Build Tests
 bin/build-test
 
+## Run Tests
 bin/run-test
+
+## Developing
+
+Make sure to clone `harmony-service-lib-py` locally
```

Comment:
Ah, yeah, I remember this from scanning this file before. I think this is an out-of-date instruction dating back to when `harmony-service-lib-py` was not [publicly available via PyPI](https://pypi.org/project/harmony-service-lib/). This dependency is listed as one of the [service Python requirements](https://github.com/nasa/harmony-gdal-adapter/blob/main/requirements.txt#L9). I think I'd suggest just removing this instruction altogether. Or, better yet, replacing it with an instruction for installing all the packages:

```
conda create --name netcdf-to-zarr python=3.9 --channel conda-forge
conda activate netcdf-to-zarr
pip install -r requirements/core.txt -r requirements/dev.txt
```

(Or alternative with instructions with `pyenv`)